tests/rptest/tests/cluster_config_test.py

```diff
@@ -169,9 +169,23 @@ class HasRedpandaAndAdmin(Protocol):
 
 class ClusterConfigHelpersMixin:
     def _check_value_everywhere(self: HasRedpandaAndAdmin, key, expect_value):
-        for node in self.redpanda.nodes:
-            actual_value = self.admin.get_cluster_config(node)[key]
-            assert actual_value == expect_value, f"Wrong value on node {node.account.hostname}: {key}={actual_value} (!={expect_value})"
+        values: dict[str, str] = {}
+
+        def _check_all():
+            nonlocal values
+            values = {
+                node.account.hostname: self.admin.get_cluster_config(node)[key]
+                for node in self.redpanda.nodes
+            }
+            self.logger.debug(f"config {values=}, {expect_value=}")
+            return all(actual_value == expect_value
+                       for _, actual_value in values.items())
```

Comment:
nit: might be simpler to use `.values()`
```suggestion
            return all(actual_value == expect_value
                       for actual_value in values.values())
```

Reply:
thanks, done